### ISSUE TITLE
Roboto Condensed variable font

### ISSUE THREAD

**Issue by emyaqin** [googlefonts/roboto-classic#119]
How can I create Roboto Condensed variable font with this source?

**Comment by kenmcd** [comment]
Roboto Condensed variable fonts are already available here:
https://fonts.google.com/specimen/Roboto+Condensed

**Comment by emyaqin** [comment]
I know its available from Google Fonts, but the font there is not updated for a long time. So there's many unsupported or tofu character. I tried with the static font from here and everything's fine, only I wish to use the variable font

**Comment by emyaqin** [comment]
I think the variable font from there is already inside the release zip here within the web folder and somehow not include character from the static one since I got a few tofu with the variable and no proble with the static

**Comment by kenmcd** [comment]
> I think the variable font from there is already included in the release zip here inside the web folder and somehow not include character from the static one since I got a few tofu with the variable and no problem with the static. I need to use the android specifically

The Android static versions include characters which are not in the other fonts (this came up in another issue recently).
But it may be possible to tweak the build scripts to add those characters to the variable font.

Because you want a variable version of the condensed font with those missing characters - is that correct?
What specific characters are missing?

Also, have you tried Roboto Flex?
It has a condensed axis available and may have your needed characters.

**Comment by emyaqin** [comment]
Yes, that's what I meant. I need to use it for my phone, I don't like the default font cause its too wide and take up too much space. I know about Roboto Flex but the default instance for that font is wide and not condensed, I don't know how to configure or build the scripts for the condensed font

**Comment by davelab6** [comment]
> How can I create Roboto Condensed variable font with this source?

Google Gemini, ChatGPT, Claude etc etc can help you use https://fonttools.readthedocs.io/en/latest/varLib/instancer.html - or you can use an app like https://slice-gui.netlify.app :)
